common/web/types/src/kmx/kmx-plus/kmx-plus.ts

```diff
@@ -381,15 +381,11 @@ export class UnicodeSetItem extends VarsItem {
 };
 
 export class SetVarItem extends VarsItem {
-  constructor(id: string, value: string[], sections: DependencySections, rawItems: string[]) {
+  constructor(id: string, value: string[], sections: DependencySections) {
     super(id, value.join(' '), sections);
     this.items = sections.elem.allocElementString(sections, value);
-    this.rawItems = rawItems;
   }
-  // element string array
-  items: ElementString;
-  // like items, but with unprocessed marker strings
-  rawItems: string[];
```

Comment:
I was going to add an underscore here ( as with `_from`  etc) but realized it was unused.